File: Carros Objective-C/Carros Objective-C/Model/Utils/Alerta.m
//
//  Alerta.m
//  Carros Objective-C
//
//  Created by Antonio Lopes on 8/5/16.
//  Copyright © 2016 Antonio Lopes. All rights reserved.
//

#import <UIKit/UIKit.h>
#import "Alerta.h"

@implementation Alerta
+(void) alerta:(NSString *)message withViewController:(UIViewController *)viewController {
    UIAlertController * alert = [UIAlertController alertControllerWithTitle:@"Alerta" message:message preferredStyle:UIAlertControllerStyleAlert];
    [alert addAction:[UIAlertAction actionWithTitle:@"Ok" style:UIAlertActionStyleDefault handler:nil]];
    
    [viewController presentViewController:alert animated:true completion:nil];
}
@end

File: Carros Swift/Carros Swift/Model/Utils/Alerta.swift
//
//  Alerta.swift
//  Carros Swift
//
//  Created by Antonio Lopes on 8/4/16.
//  Copyright © 2016 Antonio Lopes. All rights reserved.
//

import UIKit

class Alerta {
    
    class func alerta(_ message: String, viewController: UIViewController) {
        let alert = UIAlertController(title: "Alerta", message: message, preferredStyle: UIAlertControllerStyle.alert)
        alert.addAction(UIAlertAction(title: "Ok", style: UIAlertActionStyle.default, handler: nil))
        
        viewController.present(alert, animated: true, completion: nil)
    }
}
